cs/vaccination-co/index.js

```diff
@@ -249,7 +249,7 @@ function highlightFeature(e) {
     if (!L.Browser.ie && !L.Browser.opera && !L.Browser.edge) {
         layer.bringToFront();
     }
-
+	
```

Comment:
Please remove this trailing space, I will add editorconfig, the next time it will be easier for us to sync code style. https://editorconfig.org/